scripts/ci/test-examples.sh

```diff
@@ -41,17 +43,25 @@ build_local
 
 # Run Geant4 app examples
 if [ -z "${CELER_DISABLE_ACCEL_EXAMPLES}" ]; then
+  if [ -z "${G4VERSION_NUMBER}" ]; then
+    # Get the geant4 version 11.2.3, replace `.` with ` `, concat as 110203
+    _vers=$(geant4-config --version | tr '.' ' ')
+    G4VERSION_NUMBER=$(printf '%d%02d%02d' ${_vers})
+    echo "Set G4VERSION_NUMBER=${G4VERSION_NUMBER}"
+  fi
+
   # Run small accel examples
   cd "${CELER_SOURCE_DIR}/example/accel"
   build_local
   ctest -V --no-tests=error
 
-  if [ "$(echo $CELER_GEANT4_VERSION | awk -F. '{print $1*100 + $2}')" \
-      -ge 1100 ]; then
+  if [ "${G4VERSION_NUMBER}" -ge 1100 ]; then
```

Comment:
This fails with the current code at https://github.com/celeritas-project/celeritas/pull/1669/files#r1995958825
but would work with the proposed updated.  Alternatively:
```suggestion
  if [ "${G4VERSION_NUMBER}" -ge 110000 ]; then
```